src/components/Home/MostViews/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactElement, useCallback, useEffect, useState } from 'react'
+import styles from '../index.module.css'
+import { useAbortController } from '@hooks/useAbortController'
+import fetch from 'cross-fetch'
+import {
+  generateBaseQuery,
+  getFilterTerm,
+  queryMetadata
+} from '@utils/aquarius'
+import { useCancelToken } from '@hooks/useCancelToken'
+import Tooltip from '@shared/atoms/Tooltip'
+import Markdown from '@shared/Markdown'
+import AssetList from '@shared/AssetList'
+import { LoggerInstance } from '@oceanprotocol/lib'
+import { sortAssets } from '@utils/index'
+
+export default function MostViews(): ReactElement {
+  const [loading, setLoading] = useState<boolean>()
+  const [mostViewed, setMostViewed] = useState<AssetExtended[]>([])
+  const newCancelToken = useCancelToken()
```

Comment:
puh, this is fun and we should get https://github.com/oceanprotocol/market/issues/1531 done to prevent things like this, and always use `AbortController`